stock_available_unreserved/models/product_template.py

```diff
@@ -21,6 +21,7 @@ class ProductTemplate(models.Model):
     def _compute_product_available_not_res(self):
         for tmpl in self:
             if isinstance(tmpl.id, models.NewId):
+                tmpl.qty_available_not_res = False
```

Comment:
@angelgarciadelachica @etobella Really ? Should be '0.0' instead.

Reply:
If you see it better we can change it. But False is not an inappropriate value either, since in the end Odoo represents it as 0.0